grails-app/domain/org/pih/warehouse/core/Organization.groovy

```diff
@@ -28,7 +28,7 @@ class Organization extends Party {
 
     static hasMany = [locations : Location]
 
-    Map<IdentifierTypeCode, Integer> sequences
+    Map<IdentifierTypeCode, String> sequences
```

Comment:
@kchelstowski Apologies for being late to this one, but would it have been possible to leave this 
```
Map<IdentifierTypeCode, Integer> sequences
```
and change this line from
```
organization.sequences.put(IdentifierTypeCode.PURCHASE_ORDER_NUMBER.toString(), nextSequenceNumber.toString())
```
to 
```
organization.sequences.put(IdentifierTypeCode.PURCHASE_ORDER_NUMBER.toString(), nextSequenceNumber)
```